zilliqa/src/api/zil.rs

```diff
@@ -378,3 +380,34 @@ fn get_tx_block(
 
     Ok(Some(block))
 }
+
+fn get_smart_contracts(params: Params, node: &Arc<Mutex<Node>>) -> Result<Vec<SmartContract>> {
+    let address: H160 = params.one()?;
+
+    let nonce = node
+        .lock()
+        .unwrap()
+        .get_account(address, BlockNumber::Latest)?
+        .nonce;
+
+    let mut contracts = vec![];
+
+    for i in 0..nonce {
+        let contract_address = zil_contract_address(address, i);
+        // TODO: Should we expose created EVM contracts in this API too?
```

Comment:
Resolve this.

Reply:
(will leave you to close the conversation if you feel happy)

Reply:
Will remove the `TODO` and resolve, thanks!